server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java

```diff
@@ -492,8 +492,27 @@ private ColumnInfo queryTableStructure(final String tableName, String colName) {
     private String getSchema(Connection conn) throws SQLException {
         if ("h2".equalsIgnoreCase(dbType)) {
             return null;
+        } else if("postgresql".equalsIgnoreCase(dbType)) {
+            String sql = "select current_schema";
+            PreparedStatement ps = null;
+            ResultSet rs = null;
+            try {
+                conn = logStoreDataSource.getConnection();
+                ps = conn.prepareStatement(sql);
+                rs = ps.executeQuery();
+                String schema = null;
+                if (rs.next()) {
+                    schema = rs.getString(1);
+                }
+                return schema;
+            } catch (SQLException e) {
+                throw new StoreException(e);
+            } finally {
+                IOUtil.close(rs, ps, conn);
```

Comment:
conn should be closed here?

Reply:
![image](https://user-images.githubusercontent.com/24457941/106242196-3fa3e000-6242-11eb-9e53-5aa94222aab3.png)

Reply:
solved